### PR TITLE
document bulk-subscribing rails code

### PR DESCRIPTION
**Pull request by haniawni**
Document means of bulk-subscribing to known accounts for the sake of initial FTL populating on new instances

### CODE REVIEW

File: Running-Mastodon/Administration-guide.md [hunk 1]
```diff
@@ -47,6 +47,10 @@ The following rake task:
 
     RAILS_ENV=production bundle exec rails mastodon:confirm_email USER_EMAIL=alice@alice.com
 
+or, in Docker,
```
Comment (wxcafe) [inline Running-Mastodon/Administration-guide.md]: This is good but should go in a separate commit/PR

File: Running-Mastodon/Administration-guide.md [hunk 2]
```diff
@@ -67,6 +71,25 @@ only have a user record and an avatar record, with no files uploaded.
...
 
+## Bulk Subscribing to Initilly Populate FTL
+When an instance has been freshly created, it knows of very few accounts, and can only discover new ones and new servers via local users interacting with remote users. New local account entries will be constantly made as toots from currently-known accounts interact with or mention previously unknown accounts.
+
+While waiting for local users to follow remote users will gradually populate the federated timeline, initial bulk subscribing can much more rapidly discover users your community is likely to be interested in., avoiding the "deserted wasteland" feeling of a crawling FTL. Once you've used a real account to follow a handful of active, worthwhile individuals on other servers, wait for a bit to accumulate known accounts from seeing them interact. While your server knows of these newly encountered accounts, we must subscribe to them if we want them to appear in our FTL.
+
+The simplest approach is to just attempt to subscribe to every account you're not yet subscribed to. To do this, open an interactive rails console on your server:
+
+    RAILS_ENV=production bundle exec rails c
```
Comment (wxcafe) [inline Running-Mastodon/Administration-guide.md]: `rails c` should be `rails console` for clarity's sake

File: Running-Mastodon/Administration-guide.md [hunk 3]
```diff
@@ -67,6 +71,25 @@ only have a user record and an avatar record, with no files uploaded.
...
+
+or in docker:
+
+    docker-compose run --rm web rails c
```
Comment (wxcafe) [inline Running-Mastodon/Administration-guide.md]: same as above

File: Running-Mastodon/Administration-guide.md [hunk 4]
```diff
@@ -67,6 +71,25 @@ only have a user record and an avatar record, with no files uploaded.
...
+
+Then find all not-yet-subscribed-to accounts and bulk subscribe to them. THIS WILL ALMOST CERTAINLY RAPIDLY EXPAND THE SET OF KNOWN ACCOUNTS in a ballpark-exponential manner over multiple cycles, potentially resulting in excessive DB growth and FTL flooding if done too many times. Be Reasonable.
+
```
Comment (wxcafe) [inline Running-Mastodon/Administration-guide.md]: please clarify the commands below should be typed in the rails console

**Comment by wxcafe** [comment]
up? @haniawni

**Comment by nightpool** [comment]
This no longer works under ActivityPub
